src/utils/parseUserPromptOptionsFromFileProperties/parseUserPromptOptionsFromFileProperties.spec.ts

```diff
@@ -0,0 +1,322 @@
+import { FrontMatterCache } from "obsidian";
```

Comment:
This file name uses a naming convention different from the one used over the codebase (camelCase instead of kebab-case). Would it be okay with you if we rename the new files (plus `getInternalSystemPrompt.ts`) to kebab-case to make it consistent?